### DISCUSSION TITLE
Frontend & CMS Url -> broken Live Preview

### DISCUSSION THREAD

**Discussion by crackAT** [statamic/cms#6787] [Troubleshooting] [answered]
Hi all, 

our site is running with 2 different urls. One url for the page itself (frontend) and the other for the control panel access. With this settting our live preview is not working. Are there any settings to set to get the live preview running or to get the frontend running with 2 different urls? 

Best M

**Comment by jasonvarga** [comment, marked as answer]
https://statamic.dev/live-preview#headless-frontend-frameworks
